objects/rct2tt/ride/rct2.tt.barnstrm.json

```
@@ -73,6 +74,7 @@
             "sv-SE": "Passagerarna sitter i säten som hänger under spåret och färdas i hög fart genom gigantiska loopar, svängar och höga fall.",
             "ko-KR": "거대한 루프, 트위스트와 급격한 낙하 트랙을 따라 이동하며, 탑승객이 트랙 아래에 매달린 좌석에 앉는 차량",
             "zh-TW": "遊客坐在懸吊在軌道下的座椅上，當快速通過大型旋轉迴圈，令人感覺有俯衝下墜感。",
+            "ja-JP": "1920年代のダブルデッカー飛行機の形をしたインバーテッドインパルスコースターの車両です",
```

This is incorrect. Let's go with:
```suggestion
            "ja-JP": "乗客は、トラックにぶら下がった座席に座り、巨大なループ、ツイストや急降下するトラックを移動します",
```